src/config.js

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Have we considered using JSDocs typings?

Reply:
We're not thinking about that at this stage of the project yet.

Reply:
For things like this config file, it's pretty much only going to be used by us (the VBLOCKS team) right? More of an internal library kind of things, as opposed to code that the customer is going to referencing? I'm of the opinion that we should use JSDocs to make our lives easier, and that retrofitting it later down the line is going to be quite a pain. But I also understand that it's an opinionated decision. What do you think?

Reply:
The project can be run by customers locally. We will have setup instructions in the readme which include what environment variables they need to set.

We don't need JSDocs for a simple config and server file. I don't want to add something on a project that we might not need. We can always add things as we work through this project.

Reply:
I agree with keeping things slim. I do want to make sure we're on the same page in that I don't want to add any dependencies or anything, I just wanted for us to add docstrings so our IDEs and tools like VSCode can give us type hinting in these files.

Reply:
I like the idea. Here's another option. Since this is currently just a skeleton, how about we make a decision after we finish the first component and see if this is really necessary?